### ISSUE TITLE
refinement: ホームページのコンセプトを決める

### ISSUE THREAD

**Issue by ubugeeei** [vuejs-jp/home#103]

**Comment by ryoAccount** [comment]
In my opinion, I prefer dark mode to white.
The dark mode is accepted on lots of home pages related to Vue.js.
Or switching to dark and white is better too.

I wanna tell things that not only Vue.js on the home page.
That means JavaScript ecosystems.
For instance, Nuxt, Vite, Vitest, and etc are some of the technologies that we related to.

**Comment by ubugeeei** [comment]
Actually, I love dark mode for every application haha

**Comment by flyinghog88** [comment]
Agreed! I’m glad I can switch to white since I don’t like dark mode 😊️

**Comment by flyinghog88** [comment]
Here’s an idea.
How about setting up a sandbox where we can try out Vue?
